app/views/shared/_user_card.html.erb

```diff
@@ -0,0 +1,8 @@
+<ul class="photo-container">
+  <li><img src="https://th.bing.com/th/id/R.df818c6db2fe74a4230ccc863663226e?rik=Bt%2fD%2bcd60%2bv0KQ&pid=ImgRaw&r=0" alt="<%= user.name %>'s Photo" /></li>
+</ul>
+
+<ul class="card-details">
+  <li class="user-name"><%= user.name %></li>
+  <li class="card-dets-num">Number of posts: <%= user.posts_counter %></li>
+</ul>
```

Comment:
- [ OPTIONAL] Great job working on this milestone. A better approach in redering all users could be as below instead of using @objects.each (@users.each):
```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```
- Rendering a large number of objects using @objects.each can have performance implications. If you have a large collection of users or if the rendering process involves complex logic, rendering each object individually might result in a higher rendering time compared to rendering the collection as a whole.
- The @objects.each approach puts the responsibility of rendering each object on the view layer. Depending on the complexity of the rendering logic, it might result in cluttered and less maintainable view templates

Reply:
I'm confused. How do I style if I just say render @users? Assuming render users is to render all the stored users.

Reply:
- [ ] [optional] @ClintonYounge What the user was pointing out is the use of render when implementing the partial views, the stying of the page will be done in the partial view page, and incase you ave other HTML elements in your index page you can still style them.